### PR TITLE
{lang}[GCCcore/12.3.0] juliaup v1.17.9

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/358

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
Test report by @pavelToman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4001.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8
See https://gist.github.com/pavelToman/b6b80f8f47793690b7954ee07353c64d for a full test report.

**Comment by pavelToman** [comment]
Test report by @pavelToman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4204.shinx.os - Linux RHEL 9.4, x86_64, AMD EPYC 9654 96-Core Processor, Python 3.9.18
See https://gist.github.com/pavelToman/dae52d9e187a3d3980de235a2dd6400a for a full test report.

File: easybuild/easyconfigs/j/juliaup/juliaup-1.17.9-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+
+name = 'juliaup'
+version = '1.17.9'
+
+homepage = 'https://github.com/JuliaLang/juliaup/'
+description = """This repository contains a cross-platform installer for the Julia programming language."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+sources = []
```
Comment (boegel) [inline easybuild/easyconfigs/j/juliaup/juliaup-1.17.9-GCCcore-12.3.0.eb:11]:
```suggestion
sources = []
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3105.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/d0530ec5e3fa317f7c20e71659461f16 for a full test report.

**Comment by pavelToman** [comment]
Test report by @pavelToman
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3113.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.9.18
See https://gist.github.com/pavelToman/a1ec09f06ed44951dc2f628da609ea08 for a full test report.

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
> Test report by @boegel **FAILED** Build succeeded for 0 out of 1 (1 easyconfigs in total) node3105.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18 See https://gist.github.com/boegel/d0530ec5e3fa317f7c20e71659461f16 for a full test report.

It seems that their test `command_initial_setup_from_launcher_test` is broken - it expects to download ` Julia 1.11.1` but it downloads `Julia 1.11.2`. 
Since their own pipelines for this test are failing: https://github.com/JuliaLang/juliaup/commits/v1.17.9/tests/command_initial_setup_from_launcher_test.rs
I will update test to count with `Julia 1.11`

**Comment by pavelToman** [comment]
Problem reported: https://github.com/JuliaLang/juliaup/issues/1122

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
Test report by @pavelToman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3113.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.9.18
See https://gist.github.com/pavelToman/4420512e20c100bac5de797279ca6a83 for a full test report.

**Comment by pavelToman** [comment]
Test report by @pavelToman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4012.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8
See https://gist.github.com/pavelToman/e03edc116334242e18dd8df1cc072070 for a full test report.

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3127.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.11.3
See https://gist.github.com/laraPPr/53ea82317e7cc125a75875d3210cfcbf for a full test report.

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3536.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.11.3
See https://gist.github.com/laraPPr/31e638f984ad38b9d14e5f2623d4f377 for a full test report.

**Review by laraPPr — APPROVED** [review]
lgtm

**Comment by laraPPr** [comment]
All requested changes made

**Comment by laraPPr** [comment]
Going in, thanks @pavelToman!

**Pull request merged**